server/pulp/server/controllers/repository.py

```diff
@@ -763,11 +765,17 @@ def sync(repo_id, sync_config_override=None, scheduled_call_id=None):
 
     else:
         sync_end_timestamp = _now_timestamp()
+
+        added_count = model.RepositoryContentUnit.objects.num_created(
```

Comment:
Please add a plugin writer release note for RPM that added, updated, and removed counts are now calculated automatically by the platform.